movingpandas/trajectory.py

```diff
@@ -879,6 +879,37 @@ def get_sampling_interval(self):
                 return self.df[self.timedelta_col_name].median()
         return self._get_df_with_timedelta()[TIMEDELTA_COL_NAME].median()
 
+    def _get_global_velocity_length(self, units=UNITS()):
```

Comment:
This method name is confusing to me. As far as I can see, it's just the distance between start and end location. So let's call it _get_distance_between_start_and_end()